servicedirectory/quickstart_test.py

```diff
@@ -50,6 +52,7 @@ def namespace(client):
         client.delete_namespace(name=namespace.name)
 
 
+@backoff.on_exception(backoff.expo, (InternalServerError, ServiceUnavailable), max_tries=5)
```

Comment:
The `google.api_core.retry` module can already automatically retry on any transient error defined by [`is_transient_error`](https://googleapis.dev/python/google-api-core/latest/retry.html#google.api_core.retry.if_transient_error)

```py
from google.api_core.retry import Retry

@Retry
def flaky_function():
  # ...
```

The defaults are already pretty good, but you can take a look here:

https://googleapis.dev/python/google-api-core/latest/retry.html